toolkit/scss/forms/_summary.scss

```diff
@@ -109,27 +124,38 @@ table.summary-item-body {
 
 }
 
-.summary-item-field-content {
-  @extend %summary-item-field;
-  width: 66.666%;
-  padding-left: 25px;
+.summary-item-field-headings-visible {
+  @include core-16;
+  font-weight: bold;
```

Comment:
Same comment as above here, it would be good to swap out the `core-16` for `bold-16` to save this line.